stories/ag/stories.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
 import { withInfo } from '@storybook/addon-info'
-import { Theme } from '@rentpath/react-themed'
+import { Theme } from 'react-themed'
```

Comment:
I don't think this will work like this.

Reply:
I was getting a linting error, but I can put it back as it was if it works the other way.

Reply:
did you run install? the lib was recently moved to rentpath namespace

Reply:
your linter was probably failing cause it couldnt find the new dep